aya/src/bpf.rs

```diff
@@ -573,24 +577,28 @@ impl<'a> BpfLoader<'a> {
             .drain()
             .map(|(name, map)| (name, MapLock::new(map)))
             .collect();
-        Ok(Bpf { maps, programs })
+        Ok(Ebpf { maps, programs })
     }
 }
 
-impl<'a> Default for BpfLoader<'a> {
+impl<'a> Default for EbpfLoader<'a> {
     fn default() -> Self {
-        BpfLoader::new()
+        EbpfLoader::new()
     }
 }
 
+/// It is unified to eBPF because there is a loss of whether it is BPF or eBPF because it is confused.
+#[deprecated]
```

Comment:
```suggestion
/// Deprecated: `Bpf` was renamed to `Ebpf` to avoid confusion between eBPF and classic BPF.
#[deprecated(note = "Use `Ebpf` instead")]
```